frontend/src/javascript/utils/axiosUtils.js

```diff
@@ -26,3 +34,21 @@ function getBackendUrl() {
 function validateMethod(method) {
     return method && ["get", "post", "put"].includes(method.toLowerCase());
 }
+
+function getEndpointRelatedToast(method, url) {
+    if (url.startsWith("/project/find")) return { error: "Nie można pobrać projektu" };
+    if (url.startsWith("/project/findAll")) return { error: "Nie można pobrać listy projektów" };
+    if (url.startsWith("/converter/findAll")) return { error: "Nie można pobrać listy konwerterów" };
+
+    if (url.startsWith("/project/save")) return { success: "Zapisano projekt w bazie danych" };
+    if (url.startsWith("/compiler/compile"))
+        return {
+            success: "Pomyślnie skompilowano kod",
+            loading: "Trwa kompilacja...",
+            error: "Wystąpił błąd kompilacji",
+        };
+    if (url.startsWith("/converter/save")) return { success: "Zapisano konwerter w bazie danych" };
+
+    if (method.toLowerCase() == "get") return {};
+    return { success: "Pomyślnie zapisano" };
```

Comment:
Rozumiem, że to jest tak na zaś, gdyby doszły nowe endpointy to już jakiś kdomyślny komuniakt jest. Nikt w sumie nie powiedział, że postem (putem) będą szły tylko requesty zapisujące (co więcej endpoint /compiler/compile też idzie postem, więc taki komunikat dla niego trochę nie ma sensu. Zwróć tutaj pusty obiekt czy tam jakieś undefined/null co tam Ci wygodniej.